src/components/FeedbackWidget/index.tsx

```diff
@@ -31,99 +28,70 @@ const FeedbackWidget = () => {
   } = useFeedbackWidget()
   return (
     <>
-      <FixedDot
-        {...getButtonProps()}
-        onClick={handleOpen}
-        bottomOffset={bottomOffset}
-        isExpanded={isExpanded}
-      />
-
-      <AlertDialog
-        isOpen={isOpen}
-        leastDestructiveRef={cancelRef}
-        onClose={handleClose}
-        isCentered
+      <Popover
+        onOpenChange={(open) => (open ? handleOpen : handleClose)()}
+        open={isOpen}
       >
-        <AlertDialogOverlay>
-          <AlertDialogContent
-            position="fixed"
-            maxW={1504}
-            m="auto"
-            alignItems="flex-end"
-            backgroundColor="transparent"
-            me={24}
-            bottom={{ base: `${bottomOffset + 5}rem`, lg: 20 }}
-            data-testid="feedback-widget-modal"
-            padding={0}
-          >
-            <Box
-              w="min(320px, calc(100% - 1rem))"
-              mx="2"
-              bg="background.base"
-              borderRadius="base"
-              padding={{ base: "4", sm: "8" }}
-            >
-              <HStack>
-                <AlertDialogHeader fontSize="xl" fontWeight="bold" me="0">
-                  {feedbackSubmitted
-                    ? t("feedback-widget-thank-you-title")
-                    : t("feedback-widget-prompt")}
-                </AlertDialogHeader>
-                <AlertDialogCloseButton alignSelf="start" />
-              </HStack>
+        <PopoverTrigger asChild>
+          <FixedDot offsetBottom={offsetBottom} isExpanded={isExpanded} />
+        </PopoverTrigger>
+
+        <PopoverContent className="mx-2 w-80 max-w-[calc(100vw_-_1rem)] rounded bg-background p-4 sm:p-8">
+          <div className="items-top flex gap-2">
```

Comment:
`items-top` doesn't exist.

```suggestion
          <div className="flex items-start gap-2">
```

Reply:
But I want it to! 😉 Good catch

Reply:
haha! is a good name though